apps/server/src/stores/runtimeState.ts

```diff
@@ -82,7 +84,32 @@ export function getState(): Readonly<RuntimeState> {
   };
 }
 
-export function clear() {
+//TODO: find other things that dose not need clearing
```

Comment:
lets do it!

suggestion: could we have a single function resetState() which receives a patch of things to keep?

Reply:
hmmm maybe
It looks to be difficult/compute intensive if we don't just what it to be a shallow patch...
that might be ok but right now we are already keeping one deep value `_time.totalDelay`

Reply:
i dont think we should leave this comment behind as it indicates unfinished (and maybe buggy) behaviour

If we don't currently have a good idea on how to clean up lets leave as is, but rename the functions to give indication of what is happening. maybe like `clearState` and `clearEventData`?
Future us will also appreciate a comment as to what the new clear event scope is